src/struct_info.json

```diff
@@ -1668,6 +1668,28 @@
                 "sampler",
                 "textureView"
             ],
```

Comment:
While you're here would you mind updating the generator URL above to point at `main` instead of `master`?

Reply:
Done